string frame_id
bool update_frame
float32 x
float32 y
float32 z
float32 lat
float32 lon
float32 vx
float32 vy
float32 vz
float32 pitch
float32 roll
float32 thrust
float32 pitch_rate
float32 roll_rate
float32 yaw
float32 yaw_rate
---
bool success
string message
